src/renderers/shaders/ShaderChunk/fog_fragment.glsl.js

```diff
@@ -1,6 +1,12 @@
 export default /* glsl */`
 #ifdef USE_FOG
 
+	#ifdef HIGHP_PRECISION
```

Comment:
Now that #17323 has been merged I guess this can be simplified?

```
float fogDepth = precisionSafeLength(fogPosition);
```

Reply:
Definitely.

https://github.com/mrdoob/three.js/pull/17316#issuecomment-524017242

Reply:
Perfect, all done